bin/ref_arch_setup

```diff
@@ -10,6 +10,7 @@ available_commands = %w[install]
 available_subcommands = %w[generate-pe-conf bootstrap pe-infra-agent-install configure]
 
 @options = {}
+@global_options = {}
```

Comment:
these are specifically for bolt, yes?  the arg is called bolt_options in the CLI init.  

just curious why this is called global_options here.